pyi.py

```diff
@@ -752,26 +752,6 @@ def _analyze_base_node(
     {"math.inf", "math.nan", "math.e", "math.pi", "math.tau"}
```

Comment:
can't we get rid of this frozenset too?

Reply:
This is still used to check negatable defaults. I've updated the name, though.